mirdata/utils.py

```diff
@@ -17,6 +17,15 @@
     from urllib.request import urlretrieve  # py3
 except ImportError:
     from urllib import urlretrieve  # py2
+try:
```

Comment:
You can probably merge these (and all the try..excepts) from the previous blocks. Or maybe we should just do a check on the system major version and do
if sys.version.major >= 3
   do py3 imports
else
 do py2 imports